StepLang.CLI/format-config.schema.json

```diff
@@ -0,0 +1,68 @@
+﻿{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "encoding": {
+      "enum": [
+        "ansi_x3.4-1968",
+        "ansi_x3.4-1986",
+        "ascii",
+        "cp367",
+        "cp819",
+        "csascii",
+        "csisolatin1",
+        "csunicode11utf7",
+        "ibm367",
+        "ibm819",
+        "iso-10646-ucs-2",
+        "iso-8859-1",
+        "iso-ir-100",
+        "iso-ir-6",
+        "iso646-us",
+        "iso8859-1",
+        "iso_646.irv:1991",
+        "iso_8859-1",
```

Comment:
-15 could be useful here; it adds the Euro sign

Reply:
The file no longer exists